pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -788,9 +788,7 @@ def test_get_offset():
 
     pairs = [
         ("B", BDay()),
-        ("b", BDay()),
-        ("bme", BMonthEnd()),
-        ("Bme", BMonthEnd()),
+        ("BME", BMonthEnd()),
```

Comment:
`'b'` raises a deprecation warning (and it's already tested, e.g. `test_construction_bday` and `test_construction_bday`

`'bme'` would raise a hard error, but I think that's fine too, as `'me'` already went through a deprecation cycle

Reply:
yes, seems reasonable to me